ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh

```diff
@@ -55,6 +57,8 @@ cat > gather_logs.yaml <<-EOF
           LOGS_DEST: "{{ LOGS_DIR }}"
         when: kubeconfig.stat.exists
       - name: Extract capi logs
+        ansible.builtin.shell: |
+          source /root/config.sh
         make:
           chdir: /home/assisted
           target: download_capi_logs
```

Comment:
I'm not sure you can combine 2 actions in one task, something like that should work:
```suggestion
        ansible.builtin.shell: |
          source /root/config.sh
          cd /home/assisted
          make download_capi_logs
```